### ISSUE TITLE
Enhance UI of login page 

### ISSUE THREAD

**Issue by tarnibalgoher5** [Mayuresh-22/Minglr#97]

**Comment by Mayuresh-22** [comment]
hey @tarnibalgoher5, what UI changes you are planning to do?

**Comment by tarnibalgoher5** [comment]
The changes I am planning to implement are:
- There are two login buttons,vinstead have one
- The register button to be placed next to the login button
- Since it's a social networking website,a picture or animation as well can be added
- It can also be made into a small login form since it does not require too much space

**Comment by Mayuresh-22** [comment]
Let's proceed with this and we'll work on improving it throughout the development process. I will assign it to you.

**Comment by tarnibalgoher5** [comment]
okay sure

**Comment by tarnibalgoher5** [comment]
kindly assign it to me,
or should I just start working on it

**Comment by Mayuresh-22** [comment]
@tarnibalgoher5 Yeah I am assigning it to you. Start working on it.

**Comment by Mayuresh-22** [comment]
@tarnibalgoher5 are you still working on the issue?

**Comment by tarnibalgoher5** [comment]
Yes i was having some problem setting up xampp server for php. I'll complete it asap

**Comment by Mayuresh-22** [comment]
Okay, @tarnibalgoher5 if there is any unsolved problem let me know.
